packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/browser.ts

```diff
@@ -93,15 +93,9 @@ export function getBrowserConfig(wco: WebpackConfigOptions): webpack.Configurati
           vendors: false,
           vendor: !!buildOptions.vendorChunk && {
             name: 'vendor',
-            chunks: 'initial',
```

Comment:
Is initial chunks automatic now?  Only node module files from initial chunks should end up in the vendor chunk.  Otherwise the main bundle in production mode will contain all dependencies for the entire application including ones only used in lazy routes.